src/SDKs/ManagedServices/Management.ManagedServices/Microsoft.Azure.Management.ManagedServices.csproj

```diff
@@ -1,22 +1,26 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-	<!-- Please do not move/edit code below this line -->
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
-	<!-- Please do not move/edit code below this line -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code below this line -->
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
-		<Description>Microsoft Azure Management ManagedServices library</Description>
-		<AssemblyName>Microsoft.Azure.Management.ManagedServices</AssemblyName>
-		<Version>0.9.0-preview</Version>
-		<PackageTags>Azure ManagedServices;</PackageTags>
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ManagedServices</PackageId>
+    <Description>Microsoft Azure Managed Services Management Library</Description>
+    <Version>0.9.0</Version>
```

Comment:
Are you ready to publish a stable nuget version? If yes, the version must be bumped to 1.0.0

Reply:
That makes sense. I will bump it.